packages/vuetify/src/mixins/menuable/index.ts

```diff
@@ -339,7 +340,7 @@ export default baseMixins.extend<options>().extend({
       }))
     },
     updateDimensions () {
-      this.hasWindow = typeof window !== 'undefined'
+      this.hasWindow = IN_BROWSER
```

Comment:
As jacek said, also this is never going to change so it doesn't need to be updated here.

Reply:
This PR isn't about refactoring or improving existing components. Everything will be rewritten for v3 regardless.